packages/atomic/src/components.d.ts

```diff
@@ -616,6 +620,10 @@ declare namespace LocalJSX {
         "error": Error;
     }
     interface AtomicDateFacet {
+        /**
+          * The format that the date will be displayed in. See https://day.js.org/docs/en/display/format for formatting details.
```

Comment:
@jpmarceau, any preferences on how links should appear in documentation? Should they be explicit, or embedded in text?

e.g. See [Day.js](https://day.js.org/docs/en/display/format) for formatting details.

Reply:
Yes I think it would be best to use a markdown link:
see `[Day.js](https://day.js.org/docs/en/display/format)`, which we can then handle on our side to render as we want. I think this is what we do with Headless, right @kaitlynkatz ?

Reply:
Yep, it should be a markdown link so it appears embedded in text in intellisense, and then when we generate our docs we can easily use a regex to convert the link to asciidoc, as we do in Headless